routes/task.js

```diff
@@ -13,7 +13,7 @@ const authenticateRole = require("../middlewares/authenticateRole");
 const filterProjects = require("../middlewares/filterProjectsForRoles")();
```

Comment:
Found var in sendResponse function in middleware

Reply:
changed to Let!